tests/resources/service-templates/tosca-simple-1.0/node-cellar/node-cellar.yaml

```diff
@@ -98,8 +98,8 @@ topology_template:
               #token: { get_property: [ HOST, flavor_name ] }
       interfaces:
         Maintenance:
-          enable: juju > charm.maintenance_on
-          disable: juju > charm.maintenance_off
+          enable: juju \> charm.maintenance_on
```

Comment:
wait so what's this symbol about again? :)
Why are we not using the same symbol as we do for `dependencies`?